pydantic/_internal/_model_construction.py

```diff
@@ -180,6 +180,19 @@ def wrapped_model_post_init(self: BaseModel, __context: Any) -> None:
             if config_wrapper.frozen and '__hash__' not in namespace:
                 set_default_hash_func(cls, bases)
 
+            cls.__pydantic_deprecated_messages__ = {
+                field: field_info.deprecated
+                for field, field_info in cls.model_fields.items()
+                if field_info.deprecated is not None
+            }
+            cls.__pydantic_deprecated_messages__.update(
+                {
+                    field: computed_field_info.info.deprecated  # type: ignore
+                    for field, computed_field_info in cls.__pydantic_decorators__.computed_fields.items()
```

Comment:
`computed_fields` is an instance property, but that might change with https://github.com/pydantic/pydantic/pull/8437

Reply:
https://github.com/pydantic/pydantic/pull/8437 was merged, I updated this PR in the last commit to reflect changes